docs/4.2/admin-guide.md

```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
 dijkstra      c9s93fd9-3333-91d3-9999-c9s93fd98f43     10.1.0.6:3022      distro:debian
 ```
 
+### Removing Nodes from a Cluster
+
+To remove a node from a cluster, simply stop the node that you want to remove. 
+
+#### TTL time
+Node processes that are stopped have a TTL time of 10 minutes.
+You can restart the node within this time limit to keep the node. 
+
+After 10 minutes, the node is no longer considered `active` and is removed from 
+the `active SSH nodes` list within the cluster. You can  check for removable with 
```

Comment:
Double space in `can  check`

Also, this doesn't quite scan. Should be `You can check for removable nodes nodes with this command` maybe?

Reply:
Removed double space. I reworded it to `You can check that the node has been removed with
the command`. What do you think?